.github/actions/spelling/expect.txt

```diff
@@ -22,7 +22,6 @@ gvnic
 hostkeys
 iap
 ilb
-IMDS
```

Comment:
Q: Why are we removing this from spelling? I don't see it being mentioned anywhere else in this PR.